packages/zowe-explorer-api/src/profiles/ZoweExplorerZosmfApi.ts

```diff
@@ -118,6 +118,10 @@ export class ZosmfUssApi extends ZosmfApiCommon implements ZoweExplorerApi.IUss
         return zowe.Download.ussFile(this.getSession(), inputFilePath, options);
     }
 
+    public copy(outputPath: string, options?: Omit<object, "request">): Promise<Buffer> {
```

Comment:
Yeah I agree, I was thinking of typing it within ZE but it felt redundant since ultimately we are passing this `payload` object to the CLI. This at least adds temporary type safety so that only objects are passed into the `payload` parameter.